src/main/java/jarvis/logic/commands/UnmarkStudentCommand.java

```diff
@@ -0,0 +1,86 @@
+package jarvis.logic.commands;
+
+import static jarvis.commons.util.CollectionUtil.requireAllNonNull;
+import static jarvis.logic.parser.CliSyntax.PREFIX_LESSON_INDEX;
+import static jarvis.logic.parser.CliSyntax.PREFIX_STUDENT_INDEX;
+import static jarvis.model.Model.PREDICATE_SHOW_ALL_LESSONS;
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import jarvis.commons.core.Messages;
+import jarvis.commons.core.index.Index;
+import jarvis.logic.commands.exceptions.CommandException;
+import jarvis.model.Lesson;
+import jarvis.model.Model;
+import jarvis.model.Student;
+
+/**
+ * Marks a student as absent for a given lesson.
+ * The student is identified using its displayed index from the student book.
+ * The lesson is identified using its displayed index from the lesson book.
+ */
+public class UnmarkStudentCommand extends Command {
+
+    public static final String COMMAND_WORD = "markabsent";
```

Comment:
Might be better to change the command word/class name so that they are equal to be consistent?

Reply:
Was following our word doc which listed the command as markpresent and markabsent. But I think it makes sense to be consistent too. Will change it to markstudent and unmarkstudent.

Reply:
Actually I think no point cos Rachael also shortened the command words for the List Commands so some of the command words already dont match the class names